Parameter = {
    # Data frame amount for analysis
    'foot': 1,
	'forceaid': 3,
    'max_force': 900,
	'max_position': 6500,
    'zero_position' : 28500,
    'preload_position': 23500,
    'max_velocity' : 1400000,
    'nset_acc' : 400000,
    'max_pot' : 0,
    'pid_kp' : 3000.0,
    'pid_ki' : 200.0,
    'pid_umax' : 100000,
    'pid_umin' : 100000,
}
